kikuchipy/signals/util/tests/test_dask.py

```diff
@@ -38,18 +40,21 @@ def test_chunk_shape(self):
         chunks = get_chunking(s, chunk_shape=16)
         assert chunks == ((16, 16), (16, 16), (256,), (256,))
 
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="Requires Python >= 3.8")
     def test_chunk_bytes(self):
         s = LazyEBSD(da.zeros((32, 32, 256, 256), dtype=np.uint16))
         chunks = get_chunking(s, chunk_bytes=15e6)
-        assert chunks == ((8, 8, 8, 8), (8, 8, 8, 8), (256,), (256,))
+        assert chunks == ((10, 10, 10, 2), (10, 10, 10, 2), (256,), (256,))
```

Comment:
This test has the issue that it depends on how dask calculate chunks, while it shouldn't, as this is something that dask can change easily. Instead of comparing with a fixed value, would it be better to compare with the output of `dask.array.core.normalize_chunks` with the correct set of arguments?